src/lazy/expanded/compiler.rs

```diff
@@ -883,16 +988,16 @@ mod tests {
     fn expect_macro(
         definition: &TemplateMacro,
         index: usize,
-        expected_address: usize,
-        expected_num_arguments: usize,
+        expected_macro: Rc<Macro>,
+        expected_num_args: usize,
```

Comment:
🗺️ Here and below, we're updating some unit tests that previously relied on templates storing a macro address rather than holding an `Rc<Macro>`.